backend/cms/templates/pages/tree_row.html

```diff
@@ -6,9 +6,12 @@
             <i data-feather="move" class="text-grey-darkest"></i>
         </span>
     </td>
+    <td>
+        {{ node.id }}
+    </td>
     <td>
         <a href="{% url 'edit_page' page_id=node.id site_slug=site.slug language_code=language.code %}" class="block py-3 px-2 text-grey-darkest">
```

Comment:
We should remove px-2 due to alignment.

Reply:
resolved in the new version of the commit.